src/System.Management.Automation/CoreCLR/CorePsStub.cs

```diff
@@ -366,7 +371,7 @@ internal static Type GetTypeForVarEnum(VarEnum vt)
                 case VarEnum.VT_DATE:
                 case VarEnum.VT_BSTR:
 
-                // *** END GENERATED CODE ***
+                    // *** END GENERATED CODE ***
 
                     return true;
```

Comment:
seems like this line is also indented too far

Reply:
I think we can remove the comment and one above in lines 352-353.